build.gradle.kts

```diff
@@ -13,6 +13,7 @@ plugins {
 
 buildscript {
 
+    val compose_version by extra("1.1.0-beta01")
```

Comment:
Remove this because it is not being used.

Reply:
done @kibettheophilus